docs/source/blog/6-costly-data-labeling-mistakes-and-how-to-avoid-them.md

```diff
@@ -0,0 +1,127 @@
+
+Crowdsourcing can also be really helpful during this stage. If you can think back to the last time you and a group brainstormed a list of ideas, you’ll notice that group lists usually capture more information than any one person can offer. It’s the same idea here: More heads are better than one. Seek input from the other people working on the project, from the engineers to the annotators themselves. 
+
+Even with careful planning, it’s often the case that annotators will discover the need for new labels along the way. With Label Studio, administrators can give annotators the ability to add new labels while working on a task to improve accuracy.
+
+## 4. Overwhelming Tag Lists
+
+Of course, it’s also possible for project managers to arm their annotators with too many initial tags. Sifting through hundreds of tags can overwhelm your annotators, leading to longer, costlier projects. 
+
+Guided by fun heuristics like the [availability bias](https://catalogofbias.org/biases/availability-bias/), the annotators will then resort to using tags from the beginning of the list instead of scouring the entire list and finding the perfect label every time. This results in training data that doesn’t tell a complete or accurate story about the raw dataset. 
+
+Colossal tag lists also force annotators to become increasingly subjective in their choices—which can introduce more room for error. If you’re asked to label a blue square and your tag options are *red, yellow, blue,* you’ll pick the label “blue.” On the other hand, if you’re served a blue square and your tag options are *cerulean, cobalt, navy, aqua, aquamarine, royal blue, midnight blue,* you’ll have to start making decisions that might feel, to you, somewhat arbitrary. 
+
+The more shades of gray (or blue) you add to a project, the more likely you are to end up with redundant and inconsistent training data.
+
+### The Fix: Use Broad Classes With Subtasks
+
+In most cases, it’s a good idea to organize your tags around the broadest possible topics. However, there are certain projects that do require very granular labels. ImageNet famously uses [over 100,000 labels](https://image-net.org/about.php)—it doesn't just want its AI to recognize dogs; it wants it to recognize Great Danes, Dalmatians, and Water Spaniels. 
+
+If your model calls for a similar level of detail, follow the ImageNet method: Break the annotation task into smaller tasks, with each subtask representing a particular instance of the broader class. This lets annotators search through a more manageable number of broad categories before they get into the weeds. 
+
+Label Studio also has the capability to show labels dynamically as a task input, either as a prediction from the model, or from a database lookup. If you have a long list of labeling options, dynamic labeling can save annotators time and also increase the chances of selecting the best label for the object.
```

Comment:
Label Studio also can show labels dynamically as a task input, either as a prediction from the model or from a database lookup.